test/samples.js

```diff
@@ -22,6 +26,11 @@ export const validProduct = ShopApi.Product.constructFromObject({
     id: '008884303989'
 })
 
+export const validProductItem = ShopApi.ProductItem.constructFromObject({
+    product_id: validProduct.id,
+    quantity: 1
+})
+
 export const validCustomerProductListItemLink = ShopApi.CustomerProductListItemLink.constructFromObject({
     link: `https://mobify-tech-prtnr-na03-dw.demandware.net/s/2017refresh/dw/shop/v17_8/products/${validProduct.id}`,
```

Comment:
I know this is not in this PR, but can we replace the Mobify SFCC URL with the baseURL in config.json? It can save some time for the tester to switch projects.